### ISSUE TITLE
Verbosity tweaks

### ISSUE THREAD

**Issue by sjcjoosten** [utwente-fmt/vercors#233]
We would like the default verbosity of Vercors to be 'just right'. In particular:

- an expression that is not assigned to anything (like ``a + b;``) should produce a proper parse error.
- if there is an internal uncaught Java exception (in Viper or otherwise), we would like to see a stack trace (though these should not happen of course, we know they sometimes do in practice)

**Comment by niomaster** [comment]
The last point is related to https://github.com/utwente-fmt/vercors/issues/222

**Comment by Sophietje** [comment]
#227 gives an example where no output (and under some conditions a stack trace) is generated even though we would like to get some feedback from the verifier.

**Comment by Sophietje** [comment]
So Marieke found an example where an expression that is not assigned to anything does not produce an error. I minimized the example, you can find it below:
```
class KernelEncoding {
  void bla (int [] a) {
     par (int tid = 0 .. a.length)
     {
       if (tid > 0) {
         a[tid] + a[tid - 1];
         // solution: tmp = a[tid] + a[tid - 1];
       }	 
     }
  }   
}	
```
On my computer the only output is: ```[warning] missing origin```

**Comment by niomaster** [comment]
Didn't mean to step on your assigned issue, but I made some changes to the parser that may report the first error properly, we should check in https://github.com/utwente-fmt/vercors/pull/253. The second error was already fixed via #222 in https://github.com/utwente-fmt/vercors/pull/251.

**Comment by Sophietje** [comment]
No problem, I was assigned to make sure that I would add the failing examples, not to fix the issue.
